src/tools/reconciliation/csvParser.ts

```
@@ -0,0 +1,617 @@
+    // Parse description & Sanitize
+    let rawDesc = getValue(descCol)?.trim() ?? '';
+    // Security: Remove potentially malicious/confusing Unicode characters:
+    // - ASCII control chars (0x00-0x1F, 0x7F)
+    // - C1 control chars (0x80-0x9F)
+    // - Bidirectional text overrides (U+202A-202E, U+2066-2069)
+    // - Zero-width characters (U+200B-200D, U+FEFF)
+    // - Unicode line/paragraph separators (U+2028-2029)
+
+    rawDesc = rawDesc
+      // eslint-disable-next-line no-control-regex
+      .replace(/[\u0000-\u001F\u007F-\u009F]/g, '') // ASCII + C1 control chars
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)